specification/inference/_types/CommonTypes.ts

```diff
@@ -882,16 +882,27 @@ export class HuggingFaceServiceSettings {
   api_key: string
   /**
    * This setting helps to minimize the number of rate limit errors returned from Hugging Face.
-   * By default, the `hugging_face` service sets the number of requests allowed per minute to 3000.
+   * By default, the `hugging_face` service sets the number of requests allowed per minute to 3000 for all supported tasks.
+   * Hugging Face does not publish a universal rate limit — actual limits may vary.
+   * It is recommended to adjust this value based on the capacity and limits of your specific deployment environment.
    */
   rate_limit?: RateLimitSetting
   /**
    * The URL endpoint to use for the requests.
+   * For `completion` and `chat_completion` tasks, endpoint must be compatible with the OpenAI API format and include `v1/chat/completions`.
    */
   url: string
+  /**
+   * The name of the HuggingFace model to use for the inference task.
+   * For `completion` and `chat_completion` tasks, this field is optional but may be required for certain models — particularly when using serverless inference endpoints.
+   * For the `text_embedding` task, this field is not required and will be ignored if provided.
+   */
+  model_id?: string
```

Comment:
@lcawl @szabosteve In the previous docs system we created separate sections for the settings per task type. Is there a way to do that in the new docs system? What I mean is, it'd probably be easier to go to the hugging face doc page and click on a section to see all fields that are applicable for a particular task type rather than having to go through all fields and look at the text to see if it applies.